homework-4/script.js

```diff
@@ -0,0 +1,247 @@
+document.addEventListener('DOMContentLoaded', function () {
+
+  const baseUrl = 'http://localhost:3000';
+  const iconCloseBtn = document.querySelector('.icon-close-btn');
+  const headCheckbox = document.querySelector('.head-checkbox');
+  const tableBody = document.querySelector('.table-body');
+  const companyName = document.querySelector('.company-name');
+  const contactName = document.querySelector('.contact-name');
+  const address = document.querySelector('.address');
+  const city = document.querySelector('.city');
+  const country = document.querySelector('.country');
+  const usersBtn = document.querySelector('.add-users');
```

Comment:
why `add-users`? You have customers, not users :)

Reply:
change